Add getters for EventMotion properties.

hmeyer opened the pull request:
also add myself and my employer to AUTHORS.

GuillaumeGomez commented on src/event_motion.rs:
```diff
@@ -3,9 +3,24 @@
+        let y = self.as_ref().y;
+        (x, y)
+    }
+    pub fn get_state(self: Self) -> ffi::GdkModifierType {
```
missing empty line

GuillaumeGomez commented on src/event_motion.rs:
```diff
@@ -3,9 +3,24 @@
+    pub fn get_state(self: Self) -> ffi::GdkModifierType {
+        self.as_ref().state
+    }
+    pub fn get_time(self: Self) -> u32 {
```
missing empty line

GuillaumeGomez commented on COPYRIGHT:
```diff
@@ -17,8 +17,10 @@ Evgenii Pashkin <eapashkin@gmail.com>
 Geoffrey French <frondit1985@gmail.com>
 Gleb Kozyrev <gleb@gkoz.com>
 Glenn Watson <gw@intuitionlibrary.com>
+Google Inc.
```
I'm feeling uneasy with this...

gkoz commented:
@hmeyer Could you please elaborate about the role of your employer here? It doesn't seem likely that two parties at once hold copyright for this patch.

On the technical side, the getters should borrow `self` (e.g. `fn get_time(&self) -> u32`) not consume it. We try to avoid using `ffi` type names in the higher level APIs, you can use it under the `::ModifierType` name instead. The `use gdk_ffi as ffi;` line is not needed after #103.

gkoz commented:
> The use gdk_ffi as ffi; line is not needed after #103.

Or rather, this needs a rebase.

hmeyer commented:
Disclaimer: I am not a lawyer.
But as I understand it, my employer holds the copyright for all my software development work. So that name needs to go into COPYRIGHT. I consequently removed my name.
Also I addressed the other comments.

hmeyer commented:
Google employees participate in many open source projects, and Google Inc is consequently added to AUTHORS/COPYRIGHT. For example:
http://hg.mozilla.org/comm-central/file/tip/AUTHORS#l170

gkoz commented:
> as I understand it, my employer holds the copyright for all my software development work.

I'm sad to hear this. If you don't own the code there's so far no reason for us to believe it has been open-sourced by the owner.

GuillaumeGomez commented:
@hmeyer: Can you squash your commits please?

gkoz commented on src/event_motion.rs:
```diff
@@ -3,9 +3,26 @@
 event_subtype!(EventMotion, MotionNotify);
+
+impl EventMotion {
+    pub fn get_position(self: &Self) -> (f64, f64) {
```
The convention is `&self` not `self: &Self`.

gkoz commented on src/event_motion.rs:
```diff
@@ -3,9 +3,26 @@
+        (x, y)
+    }
+
+    pub fn get_state(self: &Self) -> GdkModifierType {
```
Maybe I wasn't clear, I want `ModifierType` here, it's reexported in this crate (see `lib.rs`).

hmeyer commented:
Addressed the comments (and learned how to squash commits).
COPYRIGHT: There is a LICENSE file in the project which Google Inc. is aware of and to which it agrees. Think about this: If I had just added myself to COPRYRIGHT - what reason would you have to believe I have open-sourced m the patch?

gkoz commented:
I think your employer has a policy and process for such cases designed to eliminate uncertainty. Saying that Google agrees to the license comes off as skirting that process.

hmeyer commented:
Yes, there is a process. And yes, I went through that process and got approval.

gkoz commented:
Great! Does that process allow you to paste here a copy of the approval or is it confidential?

hmeyer commented:
The approval was: "LGTM"

gkoz commented:
I guess that will do. Thanks!

GuillaumeGomez commented:
Thanks a lot for your work @hmeyer!
